squidb-android/src/com/yahoo/squidb/android/UriNotifier.java

```diff
@@ -34,25 +38,30 @@
  */
 public abstract class UriNotifier extends DataChangedNotifier<Uri> {
 
+    private final ContentResolver contentResolver;
+
     /**
      * Construct a UriNotifier that will be notified of changes to all tables
      */
-    public UriNotifier() {
+    public UriNotifier(Context context) {
```

Comment:
Should we just construct this directly with a ContentResolver instead of a Context?